Code/Correlation.py

```diff
@@ -0,0 +1,40 @@
+from scipy.stats.stats import pearsonr
+from scipy.stats import linregress
+import os
+import numpy 
+from numpy.fft import rfft, rfftfreq
+import librosa
+import matplotlib.pyplot as plt
+import librosa.display
+
+directory = 'C:/Users/BigDDY/Desktop/Test'
```

Comment:
Don't use hardcoded variables. Keep in mind that other people can want to use it in own machines.